tests/beaker_tests/cisco_bgp/test_bgp_defaults.rb

```diff
@@ -91,6 +91,13 @@
   'bestpath_med_missing_as_worst'          => 'false',
   'bestpath_med_non_deterministic'         => 'false',
   'enforce_first_as'                       => 'true',
+  'event_history_cli'                      => 'true',
+  'event_history_cli_size'                 => 'small',
+  'event_history_detail'                   => 'false',
+  'event_history_events'                   => 'true',
+  'event_history_events_size'              => 'small',
+  'event_history_periodic'                 => 'true',
+  'event_history_periodic_size'            => 'small',
```

Comment:
for test the default the expected value is just small, for non default I have changed to test all values.